presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/thrift/InMemoryHiveMetastore.java

```diff
@@ -91,10 +93,17 @@
     private final Map<PrincipalTableKey, Set<HivePrivilegeInfo>> tablePrivileges = new HashMap<>();
 
     private final File baseDirectory;
+    private final ColumnConverter columnConverter;
 
     public InMemoryHiveMetastore(File baseDirectory)
+    {
+        this(baseDirectory, new HiveColumnConverter());
+    }
+
+    public InMemoryHiveMetastore(File baseDirectory, ColumnConverter columnConverter)
```

Comment:
No need to create a new constructor given (1) it is not used else where and (2) this is a test class. Just do `private final ColumnConverter columnConverter = new HiveColumnConverter();` on the stack or make it a static one.